JavaFXUserInterface/src/main/java/AddSystemTypeController.java

```diff
@@ -72,6 +74,13 @@ public void handle(MouseEvent mouseEvent) {
                 uiUtilities.changeScene(mouseEvent, "/Systems");
             }
         });
+        //Attach link to systemTypeMenuBtn to go to SystemTypeList.fxml
+        systemTypeMenuBtn.setOnMouseClicked(new EventHandler<MouseEvent>() {
```

Comment:
All of the mouse events that are causing the scenes to be changed can be transformed into lambda expressions. This was done as part of the SonarQube refactoring for code smells. When you over the event, Intellij suggests to you. Formatting change. I will only note this change request here, but it would need to be done on all similar events that you are pushing

Reply:
All similar mouseEvent have been refactored into a lambda expression